src/workerd/api/memory-cache.c++

```diff
@@ -239,14 +239,26 @@ SharedMemoryCache::Use::~Use() noexcept(false) {
 }
 
 kj::Maybe<kj::Own<CacheValue>> SharedMemoryCache::Use::getWithoutFallback(
-    const kj::String& key) const {
-  auto data = cache->data.lockExclusive();
+    const kj::String& key, SpanBuilder& span) const {
+  kj::Locked<ThreadUnsafeData> data;
+  {
+    auto memoryCacheLockTiming =
```

Comment:
why do we need this timing virtual object at all? Why can't this MemoryCacheLockRecord be defined here in this file, and do real work?

Reply:
I think we must have been trying to avoid including timing metrics in workerd.

I removed it and make the real measurement here.